app/audit/AuditDetails.scala

```diff
@@ -18,20 +18,30 @@ package audit
 
 import cats.data.Ior
 import config.Constants
+import models.BoxId
 import models.ChannelType
 import models.EORINumber
 import models.TURN
 import play.api.libs.json.JsObject
 import play.api.libs.json.Json
 import play.api.libs.json.OWrites
 
+object AuditDetails {
+
+  def writeAuditDetails(details: AuditDetails): JsObject =
+    Json.obj(
+      "channel"    -> details.channel,
+      "customerId" -> details.customerId
+    ) ++ details.json
+}
+
 sealed abstract class AuditDetails {
   def channel: ChannelType
   def customerId: String
   def json: JsObject
 }
 
-case class AuthenticatedAuditDetails(channel: ChannelType, enrolmentId: Ior[TURN, EORINumber], json: JsObject) extends AuditDetails {
+case class AuthenticatedAuditDetails(channel: ChannelType, enrolmentId: Ior[TURN, EORINumber], json: JsObject, boxId: Option[BoxId]) extends AuditDetails {
```

Comment:
have you considered having boxId default to None?
`boxId: Option[BoxId] = None`